docs/lightning/thunderhub.md

```diff
@@ -1,5 +1,44 @@
+
 # Thunderhub
 
-GUI to manage lightning channels
+A GUI to manage lightning channels
+
+### Introduction
+
+ThunderHub is an open-source LND node manager where you can manage and monitor your node on any device or browser. It allows you to take control of the lightning network with a simple and intuitive UX.
+
+ThunderHub is a creation of Anthony Potdevin, and complete information can be found at either [Github](https://github.com/apotdevin/thunderhub) or the [ThunderHub
+website](https://thunderhub.io).
+
+![ThunderHub GUI](/docs/.vuepress/public/images/lightning/ThunderHub/ThunderHub_GUI_comp.png "ThunderHub GUI")
```

Comment:
Instead for the full address, you can just use "/images/lightning/ThunderHub/ThunderHub_GUI_comp.png"

Reply:
OK, thanks. I wasn't sure. When writing for the Wasabi docs, all you would need is "/imagename.png".
I wasn't sure that would work with our setup, so I opted for the full path.
Regardless, when I use yarn to view my page on my browser, the images do not render. Would I need to keep the image reference in my page to a local address for that to work, then change the address before pushing the page?

Reply:
Yarn should work with my suggested address format, so something else might be wrong

Reply:
Yes, it is working fine now.